CNKI.js

```diff
@@ -217,9 +228,8 @@ function scrape(ids, doc, url, itemInfo) {
 					creator.lastName = creator.lastName.substr(lastSpace + 1);
 				}
 				else {
-					// Chinese name. first character is last name, the rest are first name
-					creator.firstName = creator.lastName.substr(1);
-					creator.lastName = creator.lastName.charAt(0);
+					// 018<lyb018@gmail.com>: 20200909
+					creator.firstName = '';
```

Comment:
Why this change?

Reply:
Some people reported that Chinese names are divided into first name and last name, and only the last name is displayed in the creator column of Zotero's list. It's not easy to tell.

Reply:
>only the last name is displayed in the creator column of Zotero's list.

That's just how Zotero displays creator names in the list, no matter the language. I could see the case for an optional `Creator (full name)` column in future releases of the desktop app, but I don't think it's something we can or should address in a single translator. [See the approach we take for Baidu Scholar.](https://github.com/zotero/translators/blob/d9f2915f78f9f8c5c51c9e88bbeee94f08ad3aa5/Baidu%20Scholar.js#L133)